lib/agent.js

```diff
@@ -277,7 +278,7 @@ Agent.prototype.captureError = function (err, opts, cb) {
   if (!_isError) {
     prepareError(parsers.parseMessage(err))
   } else {
-    parsers.parseError(err, agent, function (_, error) {
+    parsers.parseError(err, captureAttributes, agent, function (_, error) {
       // As of now, parseError suppresses errors internally, but even if they
```

Comment:
I see here we're changing the function signature of the `parseError` method.  I also see elsewhere that we've updated every invocation of this method _in this package_ to use this new method signature.  We don't expose `lib/parsers` as a public api, so this method change should be ok without requiring a major version semver update.